go.mod

```diff
@@ -27,7 +27,9 @@ require (
 	github.com/j-keck/arping v1.0.0
 	github.com/json-iterator/go v1.1.6 // indirect
 	github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd
+	github.com/pkg/errors v0.8.1
```

Comment:
Have you considered creating a new Go module instead? To avoid touching Antrea's go.mod for code which is completely independent and meant to go upstream.

Reply:
we can do that if needed.

Reply:
done...

Reply:
done...

Reply:
Thanks!